adafruit_button.py

```diff
@@ -111,8 +111,8 @@ def __init__(self, *, x, y, width, height, name=None, style=RECT,
             self.selected_fill = (~self.fill_color) & 0xFFFFFF
         if self.selected_outline is None and outline_color is not None:
             self.selected_outline = (~self.outline_color) & 0xFFFFFF
-
-        if outline_color or fill_color:
+       
```

Comment:
Looks like it's failing due to the extra space added on this line.

Reply:
Ah!  I didn't even notice I had done that.  I'll delete the spaces.